test/provisioner/provision_azure.go

```diff
@@ -421,6 +421,29 @@ func (lio *AzureInstallOverlay) Delete(ctx context.Context, cfg *envconf.Config)
 
 func (lio *AzureInstallOverlay) Edit(ctx context.Context, cfg *envconf.Config, properties map[string]string) error {
 	var err error
+
+	// image
+	if AzureProps.CaaImage != "" {
+		image_and_tag := strings.Split(AzureProps.CaaImage, ":")
```

Comment:
This will fail if you specify a digest: `docker.io/library/busybox@sha256:7cc4b5aefd1d0cadf8d97d4350462ba51c694ebca145b08d7d41b41acc8db5aa`

maybe use a parsing library instead? 

https://pkg.go.dev/github.com/containers/image@v3.0.2+incompatible/docker/reference#ParseNamed

Reply:
Done